scenarios/kubernetes_janitor.py

```diff
@@ -94,6 +94,8 @@ def clean_ssh_keys(project):
     'k8s-jkns-pr-kubeadm': 3,
     'k8s-jkns-pr-kubemark': 3,
     'k8s-jkns-pr-node-e2e': 3,
+    'k8s-jkns-e2e-gke-gpus': 3,
```

Comment:
isn't this a CI project?

Reply:
weird - this project is shared between ci and pr

Reply:
This project is a CI project but I also used it for a PR job which doesn't run unless manually triggered. Is that not okay? (Didn't want to request quota for another project)

Reply:
yeah I figured that's a manual trigger job - I removed it for now